catroid/src/org/catrobat/catroid/ui/fragment/LookFragment.java

```diff
@@ -780,6 +789,30 @@ protected void showDeleteDialog() {
 		deleteLookDialog.show(getFragmentManager(), DeleteLookDialog.DIALOG_FRAGMENT_TAG);
 	}
 
+	private void moveLookDataDown() {
+		if (selectedLookPosition < lookDataList.size() - 1) {
+			Collections.swap(lookDataList, selectedLookPosition + 1, selectedLookPosition);
+			adapter.notifyDataSetChanged();
+		} else {
+			for (int i = lookDataList.size() - 1; i > 0; i--) {
+				Collections.swap(lookDataList, i, i - 1);
+			}
+			adapter.notifyDataSetChanged();
```

Comment:
> Both branches?
>>...and move it to the end of the method.

Maybe unfortunate word choice :wink:

However, since ```adapter.notifyDataSetChanged();``` is called at the end of each branch, it can be replaced by a single call by moving it to the end of the method.